### PR TITLE
Add namespaces support

### PR DESCRIPTION
**Pull request by noamokman**
Hey,

I have my alerts spread across multiple repos so I give each one an action and a namespace.
Need this for it to work :)

Thanks!

### CODE REVIEW

**Comment by noamokman** [comment]
@pracucci

**Comment by noamokman** [comment]
Also, I fixed the detailed output to support multi-line text

**Review by EdenTurgeman — APPROVED** [review]
LGTM

File: entrypoint.sh [hunk 1]
```diff
@@ -71,7 +71,10 @@ case "${ACTION}" in
 esac
 
 echo "${OUTPUT}"
-echo ::set-output name=detailed::"${OUTPUT}"
+SINGLE_LINE_OUTPUT="${OUTPUT//'%'/'%25'}"
+SINGLE_LINE_OUTPUT="${SINGLE_LINE_OUTPUT//$'\n'/'%0A'}"
+SINGLE_LINE_OUTPUT="${SINGLE_LINE_OUTPUT//$'\r'/'%0D'}"
```
Comment (pracucci) [inline entrypoint.sh:76]: I'm not sure all these single quotes are correct. For example if `OUTPUT=$(echo -e 'hello\nworld')` then the echo prints:

```
::set-output name=detailed::hello'%0A'world
```

Comment (noamokman) [reply entrypoint.sh:76]: Tested this in my repo and it works,
I copied it from this article:
https://trstringer.com/github-actions-multiline-strings/

Comment (noamokman) [reply entrypoint.sh:76]: Without it, it only prints the first line

Comment (pracucci) [reply entrypoint.sh:76]: Ok!

**Review by pracucci — APPROVED** [review]

**Comment by pracucci** [comment]
I've just pushed a new tagged release:
https://github.com/grafana/cortex-rules-action/releases/tag/v0.8.0

**Pull request merged**